DocumentDataAPI/DocumentDataAPITests/Data/Repositories/NpgWordRatioRepositoryIntegrationTests.cs

```diff
@@ -46,26 +46,26 @@ public void GetByDocumentIdAndWordReturnsCorrectWordRatio(int docID, string word
         };
 
     [Fact]
-    public void GetByDocumentIdReturnsCorrectCount()
+    public async void GetByDocumentIdReturnsCorrectCount()
```

Comment:
FFFFFFUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUU- der er mange
```suggestion
    public async Task GetByDocumentIdReturnsCorrectCount()
```